src/components/page-types/registrationFormPage/interstitialPage.js

```diff
@@ -187,14 +185,20 @@ const InterstitialPage = (props) => {
                     )}
                   </GridCell>
                 </Grid>
-                <Grid xs={12}>
+                <Grid xs={12} className="su-rs-mb-5">
```

Comment:
``
```suggestion
                <Grid gap xs={12} className="su-rs-mb-5">
```

Tip: We need the gap here so the width of each grid columns is the same as the ones with the grid below 😄 Screenshots showing the reason:
![Screen Shot 2022-07-26 at 11 23 57 PM](https://user-images.githubusercontent.com/42749717/181177214-0133839b-e5f8-4253-a6ba-6e528a152192.png)
![Screen Shot 2022-07-26 at 11 24 08 PM](https://user-images.githubusercontent.com/42749717/181177229-2f66f87b-2304-45c9-bca3-905ca27c18eb.png)